clusterloader/framework/util.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+func CreatePods(f *framework.Framework, appName string, ns string, spec v1.PodSpec, maxCount int, tuning *TuningSetType) {
+	for i := 0; i < maxCount; i++ {
+		framework.Logf("%v/%v : Creating container", i+1, maxCount)
+		labels := map[string]string{"purpose": "test"}
+		_, err := f.ClientSet.Core().Pods(ns).Create(&v1.Pod{
+			ObjectMeta: v1.ObjectMeta{
+				Name:      fmt.Sprintf(appName+"-pod-%v", i),
+				Namespace: ns,
+				Labels:    labels,
+			},
+			Spec: spec,
+		})
+		framework.ExpectNoError(err)
+		if tuning != nil {
+			if tuning.Pods.RateLimit.Delay != 0 {
+				framework.Logf("Sleeping %d ms between podcreation.", tuning.Pods.RateLimit.Delay)
+				time.Sleep(time.Duration(tuning.Pods.RateLimit.Delay) * time.Millisecond)
+			}
+			if tuning.Pods.Stepping.StepSize != 0 && (i+1)%tuning.Pods.Stepping.StepSize == 0 {
```

Comment:
id like a comment here , or at least a comment above `// CreatePods waits for blah every blah intervals`